src/main/java/com/example/DiceGameBE/service/impl/DiceService.java

```diff
@@ -1,38 +1,44 @@
 package com.example.DiceGameBE.service.impl;
 
+import com.example.DiceGameBE.exceptions.GameErrorResult;
+import com.example.DiceGameBE.exceptions.GameException;
 import com.example.DiceGameBE.model.Dice;
 import lombok.RequiredArgsConstructor;
 import lombok.extern.slf4j.Slf4j;
 import org.springframework.stereotype.Service;
-import java.util.ArrayList;
-import java.util.List;
-import java.util.Random;
+import java.util.*;
 import java.util.stream.IntStream;
 
 @Service
 @Slf4j
 @RequiredArgsConstructor
 public class DiceService {
 
-    public List<Dice> rollDice() {
-        List<Dice> diceList = new ArrayList<>();
+    public List<Dice> rollDices() throws GameException {
         Random random = new Random();
+        int numberOfRepeats = 3;
+        int numberOfRolls = random.nextInt(5) + 1;
+
+        if (numberOfRolls > 5) {
+            throw new GameException(GameErrorResult.valueOf("Number of rolls cannot > 5."));
```

Comment:
Zobacz jak to jest robione w innych przypadkach, należy dodać nowego Enuma z odpowiednim kodem i wartością. Na końcu nie ma kropek więc trzymajmy się jednej konwencji. Ponadto treść wprowadza błąd, rzuca się raz, założenie mówi o tym, że nie można rzucić więcej niż 5 kośćmi